core/src/main/scala/org/bitcoins/core/api/dlc/wallet/db/IncomingDLCOfferDb.scala

```diff
@@ -0,0 +1,18 @@
+package org.bitcoins.core.api.dlc.wallet.db
+
+import org.bitcoins.core.protocol.tlv.DLCOfferTLV
+import org.bitcoins.crypto.Sha256Digest
+
+import java.time.Instant
+
+case class IncomingDLCOfferDb(
+    hash: Sha256Digest,
+    receivedAt: Instant,
+    peer: Option[String],
+    message: Option[String],
+    offerTLV: DLCOfferTLV) {
+  require(peer.forall(_.length <= 1024),
```

Comment:
Might want to flip the max length on these. More likely to want a long message than a 1024 character tor address.